usaspending_api/disaster/v2/views/recipient/loans.py

```diff
@@ -0,0 +1,49 @@
+import json
+import re
+from decimal import Decimal
+from typing import List
+
+from usaspending_api.disaster.v2.views.disaster_base import ElasticsearchLoansPaginationMixin
+from usaspending_api.disaster.v2.views.elasticsearch_base import ElasticsearchDisasterBase
+
+
+class RecipientLoansViewSet(ElasticsearchLoansPaginationMixin, ElasticsearchDisasterBase):
+    """
+    This route takes DEF Codes and Award Type Codes and returns Loans by Recipient.
```

Comment:
Probably doesn't take award type codes for loans, yes? Or are there multiple types of loans?